adapters/syslog/syslog.go

```diff
@@ -67,7 +77,10 @@ func NewSyslogAdapter(route *router.Route) (router.LogAdapter, error) {
 	default:
 		return nil, errors.New("unsupported syslog format: " + format)
 	}
-	tmpl, err := template.New("syslog").Parse(tmplStr)
+
+	funcMap := template.FuncMap{
+		"getEnvVar":   tplGetEnvVar}
```

Comment:
Yeah, I understand. I just though it would be nice if the naming was more 'go-like'. Doesn't really matter much.
Cheers